### PR TITLE
Example networkpolicy example was wrong

### PR DESCRIPTION
**Pull request by jlin21**
Ingress.from[] takes in podSelector arrays not a map

### CODE REVIEW

**Comment by jlin21** [comment]
I signed it!

**Comment by MikeSpreitzer** [comment]
See also #832

**Comment by pwittrock** [comment]
@saad-ali Another one.

**Comment by saad-ali** [comment]
@bprashanth Can you take a look at this?

File: docs/user-guide/networkpolicies.md [hunk 1]
```diff
@@ -54,18 +54,18 @@ A minimal `NetworkPolicy` might look like this:
...
+  podSelector:
+    matchLabels:
+      role: db
  ingress:
-  - from:
-     podSelector:
-      matchLabels:
-       role: frontend
+ - from:
+    - podSelector:
```
Comment (bprashanth) [inline docs/user-guide/networkpolicies.md]: this is the only significant change right?

**Comment by bprashanth** [comment]
LGTM , you have conflicts

**Comment by jlin21** [comment]
Seems like they updated the docs

**Comment by bprashanth** [comment]
cool thanks for the pr anyway, please reopen if it needs to go in

**Pull request closed without merging**